src/test/java/org/fundacionjala/coding/manuel/Kata3Test.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.coding.manuel;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+/**
+ * Created by Administrator on 3/1/2018.
+ */
+public class Kata3Test {
+
+    private static final int TEST_NUMBER_0 = 0;
+    private static final int TEST_NUMBER_2 = 2;
+    private static final int TEST_NUMBER_3 = 3;
+    private static final int TEST_NUMBER_4 = 4;
+    private static final int TEST_NUMBER_25 = 25;
+    private static final int TEST_NUMBER_39 = 39;
+    private static final int TEST_NUMBER_444 = 444;
+    private static final int TEST_NUMBER_999 = 999;
+
+    /**
+     * test the persitance method..
+     */
+    @Test
+    public void testPersistenceKata3() {
+        Kata3 kata3 = new Kata3();
+        System.out.println("****** Basic Tests ******");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)

Reply:
Remove system.out.println